docs/0132.md

```diff
@@ -0,0 +1,167 @@
+# Rules for AEP-132: List method
+
+## List methods: No HTTP body
```

Comment:
nit: should these headers remove the `list methods` prefix, since it's already apparently in the H1?

It also might make the anchor links look better:

`/132#no-http-body`

rather than

`/132#list-methods-no-http-body`

Along the same lines, it looks like stoplight's own documentation goes with having the header name match the rule name: https://docs.stoplight.io/docs/spectral/4dec24461f3af-open-api-rules.

Which would be nice for easy referencing (just copy-paste the rule name as the anchor link and be able to navigate to it's documentation).